keyboards/nuphy/air75_v2/ansi/ansi.c

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 Persama (@Persama)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "ansi.h"
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record)
+{
+    if (!process_record_user(keycode, record)) {
+        return false;
+    }
+    switch (keycode) {
+        case MAC_VOICE:
+            if (record->event.pressed) {
+                host_consumer_send(0xcf);
+            } else {
+                host_consumer_send(0);
+            }
+            return false;
+        default:
+            return true;
+    }
+}
+
+void dial_sw_scan(void)
```

Comment:
This looks a lot like it could the dipswitch feature would cover this.

https://docs.qmk.fm/#/feature_dip_switch

Reply:
Thank you, I have replaced and modified it.